sdk/eventgrid/azure-messaging-eventgrid-cloudnative-cloudevents/README.md

```diff
@@ -47,12 +48,70 @@ and then include the direct dependency in the dependencies section without the v
 If you want to take dependency on a particular version of the library that is not present in the BOM,
 add the direct dependency to your project as follows.
 
-[//]: # ({x-version-update-start;com.azure:azure-messaging-eventgrid;current})
+[//]: # ({x-version-update-start;com.azure:azure-messaging-cloudnative-cloudevents;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
-    <artifactId>azure-messaging-eventgrid</artifactId>
-    <version>4.7.0</version>
+    <artifactId>azure-messaging-eventgrid-cloudnative-cloudevents</artifactId>
+    <version>1.0.0-beta.1</version>
 </dependency>
 ```
 [//]: # ({x-version-update-end})
+
+## Examples
+
+### Sending CNCF CloudEvents To Event Grid Topics
+```java readme-sample-sendCNCFCloudEvents
+// Make sure that the event grid topic or domain you're sending to is able to accept the EventGridEvent schema.
+List<EventGridEvent> events = new ArrayList<>();
+User user = new User("John", "James");
+events.add(new EventGridEvent("exampleSubject", "Com.Example.ExampleEventType", BinaryData.fromObject(user), "0.1"));
+eventGridEventClient.sendEvents(events);
```

Comment:
Why are we showing samples of sending EventGridEvents here? We should only have samples for sending cloud native CloudEvents.